### ISSUE TITLE
Path to a custom keychain

### ISSUE THREAD

**Issue by hdoria** [fastlane/fastlane#6342]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description

I've upgraded to macOS Sierra and since then match keeps asking me about my keychain path. Indeed, the keychain path changed on macOS Sierra, but shouldn't fastlane know about this? Am I doing something wrong?

```
To not be asked about this value, you can specify it using 'keychain_path'
Path to a custom keychain
```

### Environment

fastlane version (run `fastlane -v`):
fastlane 1.104.0

Do you use bundler to execute fastlane (i.e. `bundle exec fastlane`)? No
Do you use a Ruby environment manager (e.g. `chruby`, `rbenv`, `rvm`)? chruby

**Comment by DigitalD1982** [comment]
I'm seeing this issue as well on Sierra. I'm a first time fastlane user so I'm not sure if the problem is related to Sierra.

**Comment by DigitalD1982** [comment]
I'm wondering if this has something to do with the fact that my source code is on a different drive than my OS. I just created a test project on my master hard drive and.. It asked for the custom path to my keychain. I put in ~/Library/Keychains/login.keychain-db and everything worked great.

I then went back to my source code on my external drive and ran match development and got the error below.

[!] Could not locate the provided keychain. Tried:
	/Users/dylan/Library/Keychains/login.keychain
	/Volumes/EXTERNAL_DRIVE/Dropbox/Private/TestApps/APPNAME/login.keychain
